atlascope/core/rest/endpoints/pin_endpoints.py

```diff
@@ -8,14 +6,7 @@
 class PinViewSet(GenericViewSet):
     model = Pin
     serializer_class = PinSerializer
-    permission_classes = [IsAuthenticated]
 
     def get_queryset(self):
```

Comment:
You can probably do the same as you have done on other viewsets and simply specify `queryset = Pin.objects.all().order_by('id')` instead of `get_queryset`

Reply:
See 0458668a7bf17eac5a1eb723fcda1936121a3966.